api/models.py

```diff
@@ -2,11 +2,13 @@
 from django.utils import timezone
 from django.contrib.auth.models import User
 from django.conf import settings
+from django.dispatch import receiver
+from django.db.models.signals import post_save
 
 ACTIVITY_LEVEL_CHOICES = (
-	('SLEEPY', 'Sleepy Cat'),
-	('LAZY', 'Lazy Cat'),
+	('KITTEN', 'New Kitten'),
 	('ACTIVE', 'Active Cat'),
+	('SLEEPY', 'Sleepy Cat'),
```

Comment:
Changing models too often is generally not a good idea. Before changing something like this, always check whether the current names are used anywhere else on the site (hint: they are) and whether you're breaking anything with your new features..